theano/sparse/tests/test_basic.py

```diff
@@ -2965,7 +2965,8 @@ def structured_function(*args):
     sparse.trunc,
     np.trunc,
     test_dtypes=[m for m in sparse.all_dtypes
-                 if not m in sparse.complex_dtypes])
+                 if not m in sparse.complex_dtypes],
+    grad_test=False)
```

Comment:
We can't easily mess with the input values for sparse.

Also after some discussion, I realized that we could remove most of the tests for the sparse elemwise ops and just keep one or two to test the dense wrapper but there is no need to re-test all the ops themselves since they are tested in tensor.